pixi.js.d.ts

```diff
@@ -659,32 +665,32 @@ declare namespace PIXI {
         static TEMP_MATRIX: Matrix;
 
     }
-    export class ObservablePoint {
 
-        constructor(cb: () => any, scope?: any, x?: number, y?: number);
+    class PointLike {
```

Comment:
I think the most important benefit of the interface is being able to do stuff like `toLocal({x:1, y:42})`.